### PR TITLE
Has dedicated db

### PR DESCRIPTION
**Pull request by sedroche**
# Motivation

Ticket: https://issues.jboss.org/browse/RHMAP-15811
The current check for whether the database has been upgraded can be inaccurate.

# Solution
Add a new API that checks the database directly for the presence of the apps dedicated database.

### CODE REVIEW

File: lib/mbaasRequest/mbaasRequest.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ function _buildAdminMbaasParams(params) {
 function responseHandler(params, cb) {
 
   return function handleResponse(err, httpResponse, body) {
-    body = body || "{}";
+    body = typeof body === undefined ? "{}" : body;
```
Comment (sedroche) [inline lib/mbaasRequest/mbaasRequest.js]: I am returning a boolean here so needed to update the check. Boolean is valid JSON so should be allowed

Comment (pb82) [inline lib/mbaasRequest/mbaasRequest.js]: Or as an alternative you could return something like

```
{
  value: true
}
```

instead.

Comment (pb82) [inline lib/mbaasRequest/mbaasRequest.js]: Can `typeof body` also be `null`?

Comment (sedroche) [inline lib/mbaasRequest/mbaasRequest.js]: `null` is a valid value too. I can see your point though. i will try update.

**Pull request merged**